doc/user/content/ops/diagnosing-using-sql.md

```diff
@@ -1,24 +1,16 @@
 ---
 title: "Diagnosing Using SQL"
-description: "Use the SQL Interface to understand performance defects"
+description: "Use the SQL Interface to troubleshoot performance issues."
 menu:
   main:
     parent: operations
 ---
 
-## Diagnostic queries that can help understand apparent performance defects.
-
-Sections are meant to be thematically related based on what problems are
-experienced.
-
-The queries may be useful for spot debugging, but it may also be smart
-to create them as views and TAIL them, or as materialized views to read
-them more efficiently (understand that any of these have the potential
-to impact the system itself).
+You can use the queries below for spot debugging, but it may also be helpful to make them more permanent tools by creating them as views for `TAIL`ing or materialized views to read more efficiently. Obviously, the existence of these additional views may itself affect performance.
```

Comment:
nits, but I've been trained to try to avoid "obviously" (and similar words) as it seems to mostly shame folks who don't understand a thing. Also, it might be good to not have this as a single line? At least, the diffs will be gross if someone changes a word. No strong feels there though.

Reply:
Good point. I removed "obviously".